 sudo systemctl daemon-reexec
sudo systemctl daemon-reload
sudo systemctl restart apistudio.serviceexit
exit
\q
:Wq
python manage.py shell
exit
cd
print "hello "
cleara
clear
from django.db import connection
connection.ensure_connection()
print("✅ Connected to DB")
from django.db import connection
connection.ensure_connection()
print("✅ Connected to DB")
from django.core.management.base import BaseCommand
from django.db import connection
class Command(BaseCommand):
    help = 'Check DB connection'
    def handle(self, *args, **kwargs):
        try:
            with connection.cursor() as cursor:
                cursor.execute('SELECT 1')
                row = cursor.fetchone()
                self.stdout.write(self.style.SUCCESS(f'✅ DB connected: {row}'))
        except Exception as e:
            self.stdout.write(self.style.ERROR(f'❌ DB connection failed: {e}'))
